po/de.po

```diff
@@ -228,7 +214,7 @@ msgstr "Derzeit laufende Jobs ausgeben"
 
 #: src/builtin.cpp:392
 msgid "Evaluate math expressions"
-msgstr ""
+msgstr "Mathematische Formel ausführen"
```

Comment:
I don't think "ausführen" is correct - that's "execute".

How about "berechnen"?